### DISCUSSION TITLE
Upstreaming some improvements back to FluffyChat

### DISCUSSION THREAD

**Discussion by cyrneko** [linagora/twake-on-matrix#1799] [General]
Hey There! :wave: 

I've been testing the app quite extensively in the past couple of days to find out about the differences and changes done by Twake/Linagora to the FluffyChat codebase, things like:
- The In-App Media Picker
- Picking more than one Image/Video
- Link Previews
- "Shared media and links" in rooms and profiles
- "Show in chat" for media
- probably more...

All of these are quite good, and seem to be nicely implemented, or at least I have not found any weird edge-cases so far.

Although, being a fork of FluffyChat, what do you (as a community but also developers) think of upstreaming some of these improvements back to FluffyChat?

(Making this a public discussion to incentivize...well, discussion)

**Comment by hoangdat** [comment]
We are still open to contribute to FluffyChat, but almost is the change for UXUI. We are not sure if FluffyChat need these in its product

**Reply by cyrneko** [reply]
Generally FluffyChat welcomes PRs and upstreaming if it fits the vision and general idea, maybe you could get in contact with @krille-chan :eyes:
